### PR TITLE
[ServiceBus] b7 release doc fixes

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**
link changes to newest preview permalink, release date in changelog

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/migration_guide.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ Note: The large version gap is in order to normalize service bus SDK versions ac
...
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusSender)
-* `ServiceBusReceiver` for receiving messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.html#azure.servicebus.ServiceBusReceiver)
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusReceiver)
-* `ServiceBusSessionReceiver` for receiving messages from a session. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.html#azure.servicebus.ServiceBusSessionReceiver)
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusSessionReceiver)
+* `ServiceBusSender` for sending messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b7/azure.servicebus.html#azure.servicebus.ServiceBusSender)
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/migration_guide.md:17]: should we replace `7.0.0b7` with `latest`?

Comment (KieranBrantnerMagee) [reply sdk/servicebus/azure-servicebus/migration_guide.md:17]: Totally forgot that was an option.  Great suggestion, will save tons of work in the future, adjusting.

**Review by yunhaoling — DISMISSED** [review]
LGTM

**Review by yunhaoling — APPROVED** [review]

**Pull request merged**